vowpalwabbit/cb_explore.cc

```diff
@@ -299,8 +310,11 @@ base_learner* cb_explore_setup(options_i& options, vw& all)
   data->_random_state = all.get_random_state();
   uint32_t num_actions = data->cbcs.num_actions;
 
-  if (!options.was_supplied("cb"))
+  // If neither cb nor cats_tree are present on the reduction stack then
+  // add cb to the reduction stack as the default reduction for cb_explore.
+  if (!options.was_supplied("cats_tree") && !options.was_supplied("cb"))
```

Comment:
This could/should be moved to a cats_tree.cc

Reply:
Since cb_explore needs either cb or cats_tree on the stack, this seems like the appropriate place to make that determination.  cats_tree on the other hand is independent of cb_explore.

Does that make sense?  Can you elaborate more on moving this to cats_tree?

Reply:
This makes sense, I initially read it as cats_tree depends on this 'cb' flag but it's just setting 'cb' as a default when none is selected. Sounds good.